cmd/kubeadm/app/features/features.go

```diff
@@ -38,13 +38,16 @@ const (
 	EtcdLearnerMode = "EtcdLearnerMode"
 	// UpgradeAddonsBeforeControlPlane is expected to be in deprecated in v1.28 and will be removed in future release
 	UpgradeAddonsBeforeControlPlane = "UpgradeAddonsBeforeControlPlane"
+	// MergeCLIArgumentsWithConfig in beta in v1.29 and is expected to be removed in 1.30
+	MergeCLIArgumentsWithConfig = "MergeCLIArgumentsWithConfig"
 )
 
 // InitFeatureGates are the default feature gates for the init command
 var InitFeatureGates = FeatureList{
-	PublicKeysECDSA:      {FeatureSpec: featuregate.FeatureSpec{Default: false, PreRelease: featuregate.Alpha}},
-	RootlessControlPlane: {FeatureSpec: featuregate.FeatureSpec{Default: false, PreRelease: featuregate.Alpha}},
-	EtcdLearnerMode:      {FeatureSpec: featuregate.FeatureSpec{Default: true, PreRelease: featuregate.Beta}},
+	PublicKeysECDSA:             {FeatureSpec: featuregate.FeatureSpec{Default: false, PreRelease: featuregate.Alpha}},
+	RootlessControlPlane:        {FeatureSpec: featuregate.FeatureSpec{Default: false, PreRelease: featuregate.Alpha}},
+	EtcdLearnerMode:             {FeatureSpec: featuregate.FeatureSpec{Default: true, PreRelease: featuregate.Beta}},
+	MergeCLIArgumentsWithConfig: {FeatureSpec: featuregate.FeatureSpec{Default: false, PreRelease: featuregate.Beta}},
```

Comment:
NOTE that this is defined as `beta` instead of `alpha` since it's expected to be removed in 1.30, so `beta` might be enough.

Reply:
why not make it Deprecated FG, set to false?
see UpgradeAddonsBeforeControlPlane

Reply:
Done